<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-AVRRS09O" ignored="0" name="BankingApp Test" tests="5" failures="3" timestamp="28 Mar 2020 14:58:24 GMT" time="156.192" errors="0">
  <testcase name="getdata" time="30.093" classname="com.inetbanking.TestCases.TC_LoginDDT_02"/>
  <testcase name="getdata" time="33.973" classname="com.inetbanking.TestCases.TC_LoginDDT_02">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.TestCases.TC_LoginDDT_02.getdata(TC_LoginDDT_02.java:32)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- getdata -->
  <testcase name="getdata" time="12.949" classname="com.inetbanking.TestCases.TC_LoginDDT_02">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.TestCases.TC_LoginDDT_02.getdata(TC_LoginDDT_02.java:32)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- getdata -->
  <testcase name="getdata" time="22.631" classname="com.inetbanking.TestCases.TC_LoginDDT_02"/>
  <testcase name="getdata" time="37.897" classname="com.inetbanking.TestCases.TC_LoginDDT_02">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.TestCases.TC_LoginDDT_02.getdata(TC_LoginDDT_02.java:32)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- getdata -->
</testsuite> <!-- BankingApp Test -->
